zigpy/ota/providers.py

```diff
@@ -59,6 +59,7 @@ class BaseOtaImageMetadata(t.BaseDataclassMixin):
     max_current_file_version: int | None = None
     specificity: int | None = None
 
+    complete: bool = False
```

Comment:
This is not used yet, right?

Reply:
Oops. This was supposed to denote if the metadata for a provider is 100% complete, meaning that the image file doesn't need to be downloaded. It's not used.